frontend/components/droplets/lessons/lesson-renderer.tsx

```
@@ -337,7 +337,7 @@ function convertSingleBlock(blockAny: any, blockIndex: number): Block | null {
       const calloutContent = (blockAny.content ?? []) as any[];
       const calloutText =
         calloutContent
-          .map((item: any) => (item.type === "text" ? item.text ?? "" : ""))
+          .map((item: any) => (item.type === "text" ? (item.text ?? "") : ""))
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)